src/puya/ir/avm_ops_models.py

```diff
@@ -8,6 +8,8 @@ class StackType(enum.StrEnum):
     uint64 = enum.auto()
     bytes = "[]byte"  # noqa: A003
     bytes_32 = "[32]byte"
+    bytes_64 = "[64]byte"
+    bytes_80 = "[80]byte"
```

Comment:
Hmm, looks like we don't have any special handling of these, or even `bytes_32`. This feels like langspec is leaking - @daniel-makerx can we address this higher up and also handle the `bytes_32` case?

Reply:
Removed the bytes_* enum values from IR models, but left in the langspec models used by scripts